### ISSUE TITLE
[BUG] Office Tool Plus is not fully compatible with screen readers, needs accessibility improvements

### ISSUE THREAD

**Issue by cary-rowen** [YerongAI/Office-Tool#376]
**Office Tool Plus Version, e.g. 8.1.0.0 (20/12/2020):**
8.2.8.0

**Operating System, e.g. Windows 10 20H2 or Windows 7 SP1:**
Windows10 21H1

**Error description (required):**
Click the "Install Office" Button, the screen reader cannot read the menu item information.

*Reproduction steps (suggested):*
1. Download and install the [NVDA screen reader](https://github.com/nvaccess/nvda);
2. Click NVDA on the desktop or press Ctrl+Alt+N to open NVDA;
3. Open OTP and click the "Install Office" button;
4. Try to press the Downarrow key. NVDA can only report the "menu item", not useful information.
5. NVDA should report the title of the menu item, such as "Office Home and Student Edition 2021".

*Screenshot (suggested):*

*Additional information (optional):*
1. NVDA screen reader (free and open source, covering 175 countries around the world, is the most popular screen reader in the world): https://github.com/nvaccess/nvda
2. NVDA Chinese website is the NVDA Official Chinese Community, which gathers visually impaired users in China who use NVDA screen reader: https://nvdacn.com/

**Comment by YerongAI** [comment]
Issues fixed, the new version will release on Nov. 10.

**Comment by cary-rowen** [comment]
Thank you for your quick response, I will reopen an Issue for other questions
